vaadin-notification-flow-parent/vaadin-notification-flow/src/main/java/com/vaadin/flow/component/notification/Notification.java

```diff
@@ -521,7 +545,7 @@ public void setDuration(int duration) {
      * @return the {@code duration} property from the webcomponent
      */
     public int getDuration() {
-        return (int) getDurationDouble();
+        return (int) getElement().getProperty("duration", 0.0);
```

Comment:
```suggestion
        return getElement().getProperty("duration", 0);
```
Nit: Cast is redundant, there is an overload for ints.